### DISCUSSION TITLE
Q: Once you Start/Finish the Command Block I was wondering how it works? because ive seen other staff mods and i didnt like them, i would (if you are ok with maybe to you Spoiling it) like to know so i can have an idea of what it is

### DISCUSSION THREAD

**Discussion by historiography1945** [opekope2/StaffMod#49] [Q&A]
because I've seen other staff mods and I didn't like them, I would (if you are ok with maybe to you Spoiling it) like to know so I can have an idea of what it is.

**Comment by opekope2** [comment]
Set command, then left click/hold right click to execute it. Setting the command will be macro-able.
If you play on multiplayer, the server admin will probably limit or disable these commands.

**Reply by historiography1945** [reply]
alr thanks

**Reply by opekope2** [reply]
I opened a discussion for the Command Block Staff: https://github.com/opekope2/StaffMod/discussions/86
